### DISCUSSION TITLE
Using the transformation matrix in Biscore.spatialViewDefinition's JsonProperties

### DISCUSSION THREAD

**Discussion by Maxime-Brassard** [iTwin/itwinjs-core#6711] [Q&A]
Hello, I'm looking into a way to transform points in my 2d drawings points into 3d world points. 
In doing so, I've found this property but I've been unable to use it successfully (the "trans" property in Biscore.SpatialViewDefinition which is a 4 x 3 matrix).
My assumption was that this matrix would be a transformation matrix that, when supplied with points in drawing coordinates, would output a point in 3d world coordinates and my problem would be solved. 
I've had mixed results, it does seem like this matrix is supposed to output 3d world point but my input might not be right since it seems I always get resulting points that are very close to the origin of the spatialViewDefinition.
I was wondering if anyone knew something that could help me, thanks.

Here's a model on which you can see these properties:
https://qa-imodelconsole.bentley.com/?projectid=bd772995-9746-4e24-bac9-ba053184bce2&imodelid=e578d2e5-d901-4c70-9360-0e032916b35d&changesetid=d025ecd8d1e2e0ec5220a13bed31f287c56ea837

**Comment by pmconne** [comment]
Can you please clarify what property you are referring to?
BisCore.SpatialViewDefinition has no property named "trans", nor do we put any such property into its `jsonProperties`.
Perhaps some other application put a `trans` property in there for their own use?

**Reply by Maxime-Brassard** [reply]
I'll investigate and try to find out where the property comes from.
Will respond here as soon as I know

**Reply by Maxime-Brassard** [reply]
Me and Jonathan Decarlo talked to Chris Pair and Tim Langham and they seem to have the answers, thank you for your time.
